examples/python/controllers/controller.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import importlib
+import pandas as pd
+
+
+class Control(object):
```

Comment:
Suggest renaming to "Controller" since it is a "Controller" object.

Reply:
I changed it into "Controller"

Reply:
Thank you!